src/components/Molecules/EmailSignUp/EmailSignUp.style.js

```diff
@@ -0,0 +1,107 @@
+import styled, { css } from 'styled-components';
+import Input from '../../Atoms/Input/Input';
+import Button from '../../Atoms/Button/Button';
+import spacing from '../../../theme/shared/spacing';
+import { springScaleAnimation } from '../../../theme/shared/animations';
+import fontHelper from '../../../theme/crTheme/fontHelper';
+
+export const EmailSignUpWrapper = styled.div`
+  margin-bottom: ${spacing('m')};
+  @media ${({ theme }) => theme.breakpoints2026('M')} {
+    margin-bottom: ${spacing('m')};
+  }
+  color: ${({ theme }) => theme.color('white')};
+`;
+
+export const LabelWrapper = styled.div`
+  margin-bottom: ${spacing('md')};
+`;
+
+export const StyledLabel = styled.span`
+  ${({ theme }) => css`
+    ${fontHelper(theme, 'p')}
+    font-weight: bold;
+  `}
+`;
+
+export const FormWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: ${spacing('md')};
+  overflow: visible;
+
+
+  @media ${({ theme }) => theme.breakpoints2026('M')} {
+    flex-direction: row;
+    align-items: center;
+    ${({ error }) => error && `
+    align-items: flex-start;
+  `}
+  }
+`;
+
+export const InputWrapper = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  overflow: visible;
+  @media ${({ theme }) => theme.breakpoints2026('M')} {
+    width: auto;
+    min-width: 360px;
+  }
+`;
+
+export const StyledEmailInput = styled(Input)`
+  overflow: visible;
+
+  @media ${({ theme }) => theme.breakpoints2026('M')} {
+    ${springScaleAnimation(true, 1.04, 1)}
+  }
+
+  input {
+    ${({ theme }) => css`
+      ${fontHelper(theme, 'p')}
```

Comment:
Hm, please check with Curtis. `Input`s already have their own [styles](https://github.com/comicrelief/component-library/pull/766/changes#diff-1300026b2c85de14e0f7e73f24b3b04ba8d911177944c995c43c4e72f5ccd6efR177) laid-out - another edgecase I guess?

(this deeplink seems to be a bit wonky for me, but its at the bottom of `fontConfig.js`)